CONTRIBUTING.md

```diff
@@ -41,6 +41,38 @@ make run-mock-target
 
 Now you can point your browser to `http://localhost:3000`
 
+### Using `docker-compose`
+
+We also provide a way to develop using `docker` container and orchestrated via composition.
+
+In the `Makefile` there's a section for `DOCKER COMPOSE`:
+
+- `make down`: Will call `docker-compose down` which will stop all running containers orchestrated with our `docker-compose` configuration
+- `make build-up`: Forces the building of the different images and then boots the 3 containers
+    + back-end
```

Comment:
nit: please not that everywhere else in the project we use the `api` and `web` terms

Reply:
I'll change it to reflect that then

Reply:
@rciovati 9eb5c9375549f3e08fd9ce79050f81f9ad321549 fixes this